advanced/sr_gui_controller_tuner/src/sr_gui_controller_tuner/pid_loader_and_saver.py

```diff
@@ -90,17 +84,16 @@ def save_settings(self, param_path, parameters_dict):
 
         full_config_to_write = yaml.dump(yaml_config, default_flow_style=False)
 
-        f = open(self.path, 'w')
-        f.write(full_config_to_write)
-        f.close()
+        with open(self.path, 'w', encoding="ASCII") as pid_file:
+            f.write(full_config_to_write)
 
 
 if __name__ == '__main__':
-    path_to_config = "~"
+    PATH_TO_CONFIG = "~"
```

Comment:
Using upper case here is kind of arguable as you change the value of PATH_TO_CONFIG so it's not really a constant right? :)